data/style.css

```diff
@@ -6,6 +6,14 @@ body {
 	padding: 0 16px ;
 	margin-bottom: 500px ;
 	font-family: sans-serif ;
+	scrollbar-color: white #151515;
+}
+::-webkit-scrollbar {
+	width: .66vw;
+	background-color: #151515;
+}
+::-webkit-scrollbar-thumb {
+	background-color: white;
```

Comment:
```suggestion
	background-color: gray;
```
White is incredibly sheer.

Reply:
![Screenshot from 2021-04-23 22-30-05](https://user-images.githubusercontent.com/59586093/115931698-7aa16280-a483-11eb-91b8-9adf7180ac6d.png)
![Screenshot from 2021-04-23 22-30-22](https://user-images.githubusercontent.com/59586093/115931713-84c36100-a483-11eb-9c30-12cc2fc46b0c.png)

Reply:
Ah, yes much more pleasing. Lemme check light mode too first...